diskfs_darwin.go

```diff
@@ -0,0 +1,34 @@
+package diskfs
+
+import (
+	"fmt"
+	"os"
+
+	"golang.org/x/sys/unix"
+)
+
+// this constants should be part of "golang.org/x/sys/unix", but aren't, yet
+const (
+	DKIOCGETBLOCKSIZE         = 0x40046418
+	DKIOCGETPHYSICALBLOCKSIZE = 0x4004644D
+	DKIOCGETBLOCKCOUNT        = 0x40086419
```

Comment:
Is this `DKIOCGETBLOCKCOUNT` used anywhere?

Reply:
Nope, but in case we need that, I though would be better to let it defined

Reply:
OK, makes sense. It is a headache to track those down.